silk/__init__.py

```diff
@@ -4,3 +4,6 @@
     __version__ = get_distribution("django-silk").version
 except DistributionNotFound:
     pass
+
+# set default_app_config
+default_app_config = 'silk.apps.SilkAppConfig'
```

Comment:
isn't this deprecated?

Reply:
yes, it deprecated in Django 3.2 - but required for older version. Could only be removed once the support for > 3.2 is dropped by the package.

Reply:
you can follow this approach https://github.com/celery/django-celery-beat/blob/master/django_celery_beat/__init__.py#L34 in the new PR